src/sentry/static/sentry/app/components/inlineSvg.tsx

```diff
@@ -11,7 +11,7 @@ type Props = {
   height?: string;
 };
 
-type InlineSvgProps = Omit<React.HTMLProps<SVGSVGElement>, keyof Props> & Props;
+type InlineSvgProps = React.ComponentProps<'svg'> & Props;
```

Comment:
Was getting this error
```
00:23:41 webpack |       TS2322: Type '{ children: Element; ref: ((instance: SVGSVGElement | null) => void) | RefObject<SVGSVGElement> | null; viewBox: any; width: string; ... 356 more ...; is?: string | undefined; }' is not assignable to type 'SVGProps<SVGSVGElement>'.
00:23:41 webpack |   Types of property 'crossOrigin' are incompatible.
00:23:41 webpack |     Type 'string | undefined' is not assignable to type '"" | "anonymous" | "use-credentials" | undefined'.
00:23:41 webpack |       Type 'string' is not assignable to type '"" | "anonymous" | "use-credentials" | undefined'.
```

This is suggested via https://github.com/DefinitelyTyped/DefinitelyTyped/issues/36505

/cc @EvanPurkhiser

Reply:
Seems fine 👍